src/libraries/System.Collections/tests/Generic/OrderedDictionary/OrderedDictionary.Generic.Tests.cs

```diff
@@ -308,6 +308,25 @@ public void OrderedDictionary_Generic_SetAt_GetAt_Roundtrip(int count)
             }
         }
 
+        [Fact]
+        public void OrderedDictionary_SetAt_KeyValuePairSubsequentlyAvailable()
+        {
```

Comment:
Perhaps link back to the reporting issue?

Reply:
I'd rather not. In a few weeks the fact that there was an issue won't matter, and this test won't be any different from every other test I added.